frontend/src/screens/History/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Navbar } from '../../components';
+import useCustomContext from '../../customHooks/Hook';
+import ReactPlayer from 'react-player';
+import styles from './history.module.css';
+import { CLEAR_HISTORY, REMOVE_FROM_HISTORY } from '../../constants/type';
+
+const History = () => {
+    const { state: { history }, dispatch } = useCustomContext();
+
+    const deleteFromHistory = (id) => {
+        dispatch({ type: REMOVE_FROM_HISTORY, payload: id })
+    }
+
+    const clearHistory = () => {
+        dispatch({ type: CLEAR_HISTORY })
+    }
+
+    return (
+        <>
+            <Navbar landing={false} />
+            {
+                history.length > 0 ?
```

Comment:
&& would be helpful here right?

Reply:
Sure && is useful here, but since I have followed a convention to use ternary operator across the app therefore here also I intend to use that, to maintain consistency
Thanks